boards/arm/fk7b0m1_vbt6/board.cmake

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(openocd --target-handle=_CHIPNAME.cpu0)
+board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")
+board_runner_args(jlink "--device=STM32H7B0VB" "--speed=4000")
+
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
```

Comment:
missing a newline at the end of the file (not sure why CI doesn't catch this)

Reply:
Done. Let me know if any other modification is needed.